robot/basestation/static/js/arm.js

```diff
@@ -90,6 +83,21 @@ $(document).ready(function () {
     sendArmCommand('home') // REIMPLEMENT AS AN ACTION
   })
 
+  $('#list-all-cmds').on('click', function(event){
+    event.preventDefault()
+    printCommandsList()
+  })
+
+  $('#stop-all-motors').on('click', function(event){
+    event.preventDefault()
+    sendArmCommand('stop')
+  })
+
+  $('#reset-all-angles').on('click', function(event){
```

Comment:
In ArmExtraControls.html it uses the id `reset-motor-angles`. Up to you which id name you prefer but pls fix

Reply:
fixed